brainglobe_atlasapi/utils.py

```diff
@@ -305,14 +305,17 @@ def conf_from_url(url) -> configparser.ConfigParser:
     text = requests.get(url).text
     config_obj = configparser.ConfigParser()
     config_obj.read_string(text)
-    cache_path = config.get_brainglobe_dir() / "last_versions.conf"
+    cache_path: Path = config.get_brainglobe_dir() / "last_versions.conf"
 
-    if not cache_path.parent.exists():
-        cache_path.parent.mkdir(parents=True, exist_ok=True)
-
-    # Cache the available atlases
-    with open(cache_path, "w") as f_out:
-        config_obj.write(f_out)
+    try:
+        if not cache_path.parent.exists():
+            cache_path.parent.mkdir(parents=True, exist_ok=True)
+
+        # Cache the available atlases
+        with open(cache_path, "w") as f_out:
+            config_obj.write(f_out)
+    except OSError as e:
+        print(f"Could not update the latest atlas versions cache: {e}")
```

Comment:
Would it be better if this used `logging.warning` or `logging.info`?

Reply:
I wasn't sure, the rest of the file uses `print` statements to inform user of error cases so I didn't want to break from what's currently being used. Does `brainglobe-atlasapi` keep a log on disk somewhere?

Reply:
It looks like earlier in the `utils.py` file it sets the logging level for `urllib3` but if `print` is used elsewhere, maybe let's stick to `print` here and open an issue to make logging consistent... (`atlasgen` uses `loguru`)